docs/reference/permission_matrix.rst

```diff
@@ -0,0 +1,28 @@
+The permission matrix
+=====================
+
+Configuration
+-------------
+
+The type of form ``sonata_security_roles`` was improved to show all roles in matrix.
```

Comment:
please us FQCN instead of `sonata_security_roles`

Reply:
`Sonata\UserBundle\Form\Type\SecurityRolesType`

Reply:
FQCN = fully qualified class name

Reply:
`Sonata\UserBundle\Form\Type\SecurityRolesType` is QCN. FQCN should be `\Sonata\UserBundle\Form\Type\SecurityRolesType`.

Reply:
without the initial / then